src/ImageSharp/Metadata/Profiles/Exif/ExifConstants.cs

```diff
@@ -22,5 +36,55 @@ internal static class ExifConstants
             0x00,
             0x2A
         };
+
+        public static Encoding DefaultAsciiEncoding => Encoding.UTF8;
```

Comment:
```suggestion
        public static Encoding DefaultAsciiEncoding => Encoding.Ascii;
```
? Or adjust the property name.

Reply:
>but in fact the code use UTF

What code, ours or a reference implementation?

Reply:
I think, UTF8 is better than ASCII:  
https://stackoverflow.com/questions/19284205/safe-to-use-utf8-decoding-for-exif-property-marked-as-ascii/19284524#19284524

Reply:
Thanks for the info, yeah, the spec let's everyone down there. Shall we rename the constant then. `DefaultEncoding` and add a comment stating why we use UTF8.